data/intelligence_tag_metadata.yaml

```diff
@@ -0,0 +1,15 @@
+malware:
```

Comment:
Can we add a description of the format i this file? Explain what weight and class means for example.

Reply:
Done.